homeassistant/components/xiaomi_aqara/binary_sensor.py

```diff
@@ -293,8 +293,8 @@ def parse_data(self, data, raw_data):
                 )
 
             self._no_motion_since = 0
-            if self._state:
-                return False
+#            if self._state:
+#                return False
```

Comment:
If you have a valid reason for this code to be removed, please address this in the description of your pull request. Tests will also likely need to be adjusted